src/coda/apps/fundingrequests/views/review.py

```diff
@@ -45,16 +46,16 @@ def process_review(review: Review, request: HttpRequest) -> Review:
     action = request.POST["action"]
     match action:
         case "approve":
-            review = review.approved(funding, remarks)
+            review = review.update_review(ReviewResult.Approved, funding, remarks)
         case "reject":
-            review = review.rejected(remarks)
+            review = review.update_review(ReviewResult.Rejected, funding, remarks)
         case "close":
-            review = review.closed(remarks)
+            review = review.update_review(ReviewResult.Closed, funding, remarks)
         case "waive":
-            review = review.costs_waived(remarks)
+            review = review.update_review(ReviewResult.Waived, funding, remarks)
         case "open":
-            review = review.opened(remarks)
+            review = review.update_review(ReviewResult.Open, funding, remarks)
         case "return":
-            review = review.with_remarks(remarks)
+            review = review.update_review(decided_funding=funding, remarks=remarks)
```

Comment:
Now that we don't use separate methods for review state changes anymore, please create a ReviewResult directly from the action string to avoid the long match statement